shotover-proxy/tests/cassandra_int_tests/batch_statements.rs

```diff
@@ -9,12 +8,15 @@ pub async fn test(connection: &CassandraConnection) {
     }
 
     {
-        let mut batch = Batch::new(BatchType::LOGGED);
+        let mut queries: Vec<(String, i32)> = vec![];
         for i in 0..2 {
-            let statement = format!("INSERT INTO batch_keyspace.batch_table (id, lastname, firstname) VALUES ({}, 'text1', 'text2')", i);
-            batch.add_statement(&stmt!(statement.as_str())).unwrap();
+            queries.push((
+                "INSERT INTO batch_keyspace.batch_table (id, lastname, firstname) VALUES (?, 'text1', 'text2')".into(),
```

Comment:
I dont understand why we are changing execute_batch to perform binding?
Arent we better off just using plain old queries?

Reply:
Hmm I had assumed you couldn't use it that way with `cdrs-tokio` but I might be wrong. Will have a look into that.